### PR TITLE
[14.0][MIG] Account_move_line_stock_info

### PR DESCRIPTION
**Pull request by JoanMForgeFlow**
Standard migration to v.14
@mariadforgeflow
@bernatpforgeflow

### CODE REVIEW

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Review by rousseldenis — APPROVED** [review]
code review

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    def test_account_move_line_stock_move(self):
+        """Test that processing an incoming picking the account moves
+        generated by the picking moves will contain the stock move.
+        """
+        picking_in = self._create_picking(
+            self.picking_type_in, self.location_supplier, self.location_stock
+        )
+        picking_in.action_confirm()
+        picking_in.move_lines.quantity_done = 1
+        picking_in.action_done()
```
Comment (rousseldenis) [inline account_move_line_stock_info/tests/test_account_move_line_stock_move.py]:
```suggestion
        picking_in._action_done()
```

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            self.assertEqual(len(move.account_move_line_ids), 2)
+            for aml in move.account_move_line_ids:
+                account_move_line = aml
+
+        picking_out = self._create_picking(
+            self.picking_type_out, self.location_supplier, self.location_stock
+        )
+        picking_out.action_confirm()
+        picking_out.move_lines.quantity_done = 1
+        picking_out.action_done()
```
Comment (rousseldenis) [inline account_move_line_stock_info/tests/test_account_move_line_stock_move.py]:
```suggestion
        picking_out._action_done()
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: account_move_line_stock_info/tests/test_account_move_line_stock_move.py (hunk 2 above)
Comment (rousseldenis) [reply account_move_line_stock_info/tests/test_account_move_line_stock_move.py]: Missed that

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional review. Pending to fix CI

**Comment by AaronHForgeFlow** [comment]
@rousseldenis it seems CI is ok now :)

**Comment by JoanMForgeFlow** [comment]
@rousseldenis can you update your review please?

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**